src/service/api.tsx

```diff
@@ -134,7 +138,47 @@ const api = {
     return obj.data.results;
   },
 
-  changeImgLinks(url: string, size: string) {
+  async getSearchFileredList(dataFilter:
+    {
+      from: number,
+      to: number,
+      genre: string | null,
+      page: number
+    } = {
+      from: 1900,
+      to: 2020,
```

Comment:
Do not use magic numbers - this should be moved to const variables
2020 - I believe should be converted to new Date().getFullYear() to be dynamic

Reply:
no, here logic based on only years and not need Date().getFullYear() for that UI

Reply:
ok, what these numbers used for then?

Reply:
These numbers are ids of genres

Reply:
if so why it is being used as date? `&primary_release_date.gte=${from}-01-01&primary_release_date.lte=${to}-01-01`